identity/res/values/strings.xml

```diff
@@ -153,8 +167,14 @@
   <string name="stripe_position_passport">Position your passport in the center of the frame</string>
   <!-- Instructional text for capturing selfie -->
   <string name="stripe_position_selfie">Position your face in the center of the frame.</string>
+  <!-- Instructional text when the document is too blurry -->
+  <string name="stripe_reduce_blur">ID not in focus</string>
+  <!-- Instructional text when the document has too much glare -->
+  <string name="stripe_reduce_glare">Details not visible</string>
   <!-- Button text displayed to the user to retake photo -->
   <string name="stripe_retake_photos">Retake Photos</string>
+  <!-- Instructional text when user needs to rotate the document to align with camera -->
+  <string name="stripe_rotate_document">Kee ID level</string>
```

Comment:
Found a typo. Let’s request a fix on Lokalise before we merge this.

Reply:
Pushed a fix for this typo and submitted a ticket about the faulty text. It should be fixed by next Monday when we do the next automated translation update.